internal/lsp/server.go

```diff
@@ -453,36 +462,36 @@ func (l *LanguageServer) handleInitialize(
 		},
 	}
 
-	folderURI, err := url.Parse(l.clientRootURI)
-	if err != nil {
-		return nil, fmt.Errorf("failed to parse URI: %w", err)
-	}
+	workspaceRootPath := uri.ToPath(l.clientIdentifier, l.clientRootURI)
+	l.logError(fmt.Errorf("workspaceRoot: %s", workspaceRootPath))
```

Comment:
Or wait, you're logging at error level here too. Is that all we got?

Reply:
we have log too, it goes to std err, since std in is reserved for the JSON RPC data.

Reply:
This isn't needed so have removed in 1d516dcaacc3aecb3cf1640c555d1c11fc2c8ca9